pytest_timeout.py

```diff
@@ -383,14 +428,25 @@ def _validate_func_only(func_only, where):
     return func_only
 
 
-def timeout_sigalrm(item, timeout):
+def _validate_disable_debugger_detection(disable_debugger_detection, where):
+    if disable_debugger_detection is None:
+        return None
+    if not isinstance(disable_debugger_detection, bool):
```

Comment:
hum, this escaped me for `_validate_func_only()` it seems, but the pythonic thing is to call `bool(disable_debugger_detection)` and just go with that.  `_validate_timeout()` also goes with `float(...)`.

(if you fix up `_validate_func_only()` that's be nice as well, i think that'd be a backwards-compatible change)

Reply:
Definitely simpler, but runs into some annoying cases with string truthyness and booleans. i.e. Typing out `false` instead of `False` in the ini/env vars would give True.

For `_validate_timeout`, is it OK to get rid of the `where` information in the error message?

Reply:
Ok, I don't mind if we leave this as-is either.  It's already how it works and no-one has complained about this so far, I think.

Reply:
I'll keep it as is.